apispec/asyncapi-circe/src/main/scala/sttp/tapir/asyncapi/circe/package.scala

```diff
@@ -155,4 +159,14 @@ trait TapirAsyncAPICirceEncoders {
       val properties = m.mapValues(v => implicitly[Encoder[V]].apply(v)).toList
       Json.obj(properties: _*)
   }
+
+  private def expandExtensions(jsonObject: JsonObject): JsonObject = {
```

Comment:
maybe you could add a comment on what's the JSON transformation that is happening there, and why is it needed?

Reply:
Sure. Done